content/practices/outcome-oriented-roadmap/index.md

```diff
@@ -0,0 +1,225 @@
+1. Check the First Draft of the Roadmap, and Adjust as Needed (10 minutes)
+   
+   **Discuss** using the following prompts:
+   - Does this roadmap help us move towards our vision?
+   - Are these the highest value outcomes for us to work on first?
+   
+   If answers are not a clear yes, which adjustments do team members suggest? Repeat step 6–7 as needed.
+   
+   Don’t forget to celebrate: your team has built an iteration of its outcome-oriented roadmap! 🎉
+   
+1. After this Session
+   
+   **On a regular basis**, review your progress through your roadmap by asking the team to roman vote their progress through each outcome. 
+   
+   - Thumbs up - We’ve achieved the outcome!
+   - Middle - We’re still working on this outcome.
+   - Thumbs down - We’ve not started to work on this outcome or it’s no longer relevant.
+   
+   This is to align on the team’s assessment of your progress. For outcomes that have been voted middle or thumbs down, action items need to be defined (e.g. break down outcome, focus better, revisit relevancy).
+   
+   Alternatively, the team could review the metrics and assess whether the outcomes have been achieved. 
+   
+   Find a way for your team to capture the progress for each outcome - this could be done via color coding (e.g. green, orange, red), by using progress bars (e.g. 75% achieved), or by whatever works for you.
+   
+   **Following this alignment session, update your roadmap.** You can do that by repeating steps 3–8.
+   
+   {{% callout %}}
+   Tip 1: Block out time on a regular cadence (~ monthly) to review your roadmap to keep your roadmap up to date. Also make your roadmap visible for everyone: Either on the wall in your office or prominently in a remote working board to be reviewed during Iteration Planning Meetings, internal and stakeholder updates. 
+   {{% /callout %}}
+   
+   {{% callout %}}
+   Tip 2: Less mature product ideas often may require more frequent updating, as outcomes are validated or invalidated, and there is generally less certainty about what is right to do next. Do not worry if you have to update and change this regularly!   
+   {{% /callout %}}
+
+
+#### Success/Expected Outcomes
+At the end of this exercise, you will have a roadmap of product organized sequentially in this way:  
+
+![Now, Near Term, and Future Term Outcomes](/images/practices/outcome-oriented-roadmap/now-near-future-1.png)
+
+#### Facilitator Notes & Tips
+MVP: The **Minimum Viable Product** (MVP) is an experiment designed to test our product’s value proposition. It’s the **smallest possible version of our product** that enables us to learn the most about our customers with the least amount of effort.
+
+Additional resources:
+- [SVP: The Shoddiest Viable Product](https://tanzu.vmware.com/content/blog/svp-the-shoddiest-viable-product)
+- [Why "MVP" is Confusing](https://tajmo.medium.com/why-mvp-is-a-contronym-40af0fcb74c0)
+
+## Related Practices
+
+### Preceding
+Vision, mission, strategy -- coming soon!
```

Comment:
Are these practices, workshops, or stages in the project lifecycle? If not practices / workshops we can remove them from this list and describe vision, mission, and strategy elsewhere.